src/c/user-settings.js

```diff
@@ -0,0 +1,435 @@
+import m from 'mithril';
+import models from '../models';
+import postgrest from 'mithril-postgrest';
+import _ from 'underscore';
+import h from '../h';
+import userVM from '../vms/user-vm';
+import inlineError from './inline-error';
+import projectCard from './project-card';
+
+const userSettings = {
+    controller(args) {
+        const user = args.user,
+              fields = {
```

Comment:
We should probably have those fields inside a VM. Those are actually ready to use on the payment-vm, but I don't know if there's where they should be. Maybe we could discuss this a bit further together with the team.